components/python/cheroot/history

```diff
@@ -1 +1,2 @@
+library/python/cheroot-27@6.5.2-2020.0.1.5
 library/python/cheroot-34@6.5.2-2020.0.1.2
```

Comment:
Bot lines missing ",5.11", like
library/python/cheroot-27@6.5.2,5.11-2020.0.1.5

Reply:
Again we should check but I was pretty sure we modified pkg to support short fmris...